src/components/app-bar/AppBar.svelte

```diff
@@ -43,44 +38,32 @@
 		polkadotSigner.set(account.polkadotSigner);
 		setActiveAccountBounties();
 	});
-
-	async function logOut() {
-		activeAccount.set(undefined);
-		sessionStorage.clear();
-		activeAccountBounties.set([]);
-		await $walletConnect?.disconnect();
-	}
 </script>
 
-<header
-	class="relative flex items-center justify-between h-20 bg-primary px-4 sm:px-12 overflow-clip"
->
-	<div class="md:absolute md:left-1/2 md:transform md:-translate-x-1/2 pt-2">
-		<a href="/curator-actions" class="hidden md:inline-flex">
-			<img width="307" height="62" src={LogoBountyManagerDesktop} alt="Logo" />
-		</a>
-		<a href="/curator-actions" class="md:hidden">
-			<img width="71" height="54" src={LogoBountyManagerMobile} alt="Logo" />
+<header class="relative flex items-center justify-between min-h-20 bg-primary px-4 sm:px-12">
+	<div>
+		<a href="/curator-actions">
+			<img width="71" height="54" src={LogoBountyManagerHeader} alt="Logo Bounty Manager" />
 		</a>
 	</div>
 
-	<div class="ml-auto">
+	<div>
 		{#if !$activeAccount}
 			<button class=" text-white" on:click={showLoginDialog}>Connect Wallet</button>
 
 			<w3m-button></w3m-button>
 		{:else}
-			<div class="flex items-center space-x-3">
+			<!-- User Address -->
+			<div class="flex items-center align-top space-x-3">
 				<div class=" flex gap-2 items-center text-white">
 					<div class="w-6 h-6">
 						<PolkadotIcon address={$activeAccount.address} />
 					</div>
 					{$activeAccount.name || 'Account'}
 					<span class="text-darkgray text-sm">[{truncateString($activeAccount.address, 4)}]</span>
 				</div>
-				<button on:click={() => logOut()} class="mt-2">
-					<span class="material-symbols-outlined text-white text-2xl">logout</span>
-				</button>
+				<!-- Burger Menu -->
+				<BurgerMenu />
```

Comment:
```suggestion
				<BurgerMenu />
```
Is this comment helpful? :)